examples/yesod-pubsub/stack.yaml

```diff
@@ -0,0 +1,72 @@
+# This file was automatically generated by 'stack init'
```

Comment:
we don't really need local stack file, don't we ?

Reply:
Deleted